### DISCUSSION TITLE
Does this work with:

### DISCUSSION THREAD

**Discussion by jipvankuijk** [johannrichard/homebridge-dingz#334] [Q&A] [answered]
Hello, does this work with the following motion sensor?

I'd like to use it in Homekit via Homebridge.

Product: https://mystrom.ch/wifi-motion-sensor/

**Comment by johannrichard** [comment, marked as answer]
Yes, it works with the motion sensors as well, have a look in the [Wiki](https://github.com/johannrichard/homebridge-dingz/wiki#dingz-devices).
